salt/modules/linux_lvm.py

```diff
@@ -216,9 +216,11 @@ def pvcreate(devices, override=True, **kwargs):
     '''
     if not devices:
         return 'Error: at least one device is required'
+    if isinstance(devices, str):
```

Comment:
It will be better here to use `six.string_types` here instead of `str`, I think.